v1/src/main/java/com/google/cloud/teleport/templates/common/SpannerConverters.java

```diff
@@ -322,7 +398,7 @@ private String createColumnExpression(String columnName, String columnType, Dial
           return "CAST(`" + columnName + "` AS STRING) AS " + columnName;
         }
         if (columnType.equals("JSON")) {
-          return "TO_JSON_STRING(`" + columnName + "`) AS " + columnName;
+          return "`" + columnName + "`";
```

Comment:
Follow up on impact to existing customers.

Reply:
It won't create a  difference because while writing it to csv, we are converting JSON back to String.

Reply:
Have you tested the CSV export with this change? I thought we always need TO_JSON_STRING in the query while reading json columns.